docs/api/native-image.md

```diff
@@ -119,6 +119,13 @@ Returns `NativeImage`
 
 Creates an empty `NativeImage` instance.
 
+### `nativeImage.createThumbnailFromPath(path, maxSize)` _macOS_ _Windows_
+
+* `path` String - path to a file that we intend to construct a thumbnail out of.
+* `maxSize` [Size](structures/size.md) - the maximum width and height the thumbnail returned can be. The Windows implementation will ignore this parameter and scale the height according to `width`.
```

Comment:
nits:

* Should document that `width` and `height` must be positive

* Guessing without peeking: this sounds like we're saying that `maxSize.width` *will* be honored? If so, we shouldn't say Windows will ignore `maxSize` and instead say it will ignore `maxSize.height`

* Similar to previous, "...according to `width`" would be marginally clearer as "...according to `maxSize.width`"